src/components/existing-request/existing-request-display.vue

```diff
@@ -749,4 +901,36 @@ export default class ExistingRequestDisplay extends Mixins(
     font-weight: bold;
   }
 }
+
+.v-tooltip__content {
+  width: 24rem !important;
+  text-align: justify;
+  text-justify: inter-word;
+}
+
+.v-tooltip__content:after {
+  content: "" !important;
+  position: absolute !important;
+  top: 100% !important;
+  right: 50% !important;
+  border-right: 10px solid transparent !important;
+  border-left: 10px solid transparent !important;
+  border-top: 8px solid RGB(73, 80, 87) !important;
+  transform: translateY(50%);
+}
```

Comment:
I believe the tool tips were styled in the overrides already. Does this design call for something different than we've already done?

Check it out though, there is tool tips for top, bottom and right i think,

Reply:
They actually asked me to use a tooltip with 270px as it is the default but it doesn't look good:
![image](https://user-images.githubusercontent.com/78114138/129987750-0c27ee4e-d0d4-4bf2-a444-fe297edea0d8.png)

the `:after` is to make the arrow be pointed down.

Reply:
I understand the context, this style already exists (or close too) in `overrides.scss` 
You would need to add the content class to your tool tip, you might not need to duplicate the styling in the component itself. 

See class `top-tooltip` in overrides. Using this on the tool tip itself is by adding it as a content class.
`<v-tooltip top content-class="top-tooltip">`

If you need to override the width that can be set on the component itself as well.

Reply:
ok. Got it. Thanks!

Reply:
Awesome!